### ISSUE TITLE
some system extensions require machine config changes (e.g. loading a kernel module), can we automate it?

### ISSUE THREAD

**Issue by Unix4ever** [siderolabs/omni#128]

**Comment by rothgar** [comment]
Yes please! This would be great to have the nvidia-kmod extension automatically know to patch the node to include kernel modules too.

**Comment by stereobutter** [comment]
Would this make sense as a general talos feature (instead of an omni feature). Maybe add a field e.g. `additionalMachinePatches` to the spec of `manifest.yaml` and have talos apply those during the install phase when extensions with the field set are present.
